xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/java/org/xwiki/officeimporter/script/OfficeImporterScriptService.java

```diff
@@ -327,7 +327,7 @@ public boolean save(XDOMOfficeDocument doc, DocumentReference documentReference,
                 String currentSyntaxId = this.docBridge.getDocument(documentReference).getSyntax().toIdString();
                 if (!currentSyntaxId.equals(syntaxId)) {
                     String message =
-                        "Target document [%s] exists but it's syntax [%s] is different from specified syntax [%s]";
+                        "Target document [%s] exists but its syntax [%s] is different from specified syntax [%s]";
```

Comment:
"Target document Main.Test exists but its syntax [2.0] is different from specified syntax [2.1]"
vs.
"The target document Main.Test exists but its syntax [2.0] is different from the specified syntax [2.1]"

Sounds good. @vrachieru thanks for the suggestion. Can you make a PR? :)